app/validators/email_address_validator.rb

```diff
@@ -0,0 +1,16 @@
+class EmailAddressValidator < ActiveModel::EachValidator
+  include ValidatorConcerns
+  EMAIL_FORMAT = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
```

Comment:
Not for now, but I'm wondering if we can re-use this email validator for the contact_email field in the job posting eventually. We have a ton of validators all over the place and it might be nice to reconcile them a bit.